zerver/openapi/zulip.yaml

```diff
@@ -9787,6 +9797,9 @@ paths:
                             - `URL` for links.
                             - `EXTERNAL_ACCOUNT` for external accounts.
                             - `USER` for selecting a user for the field.
+                            - `PRONOUNS`
```

Comment:
Not sure how to explain this type in a short and concise way.

Reply:
`PRONOUNS` for a short text field that makes it easy to enter one's pronouns.